editor/src/components/canvas/controls/select-mode-control-container.tsx

```diff
@@ -104,13 +106,28 @@ export class SelectModeControlContainer extends React.Component<
       if (isMultiselect) {
         updatedSelection = TP.addPathIfMissing(target, this.props.selectedViews)
       }
-      let selectActions = [
-        EditorActions.clearHighlightedViews(),
-        EditorActions.selectComponents(updatedSelection, false),
-        EditorActions.setLeftMenuTab(LeftMenuTab.UINavigate),
-        EditorActions.setRightMenuTab(RightMenuTab.Inspector),
-      ]
-      this.props.dispatch(selectActions, 'canvas')
+
+      ;(ReactDOM as any).flushSync(() => {
```

Comment:
If this is secret functionality could you add a comment indicating what this is up to please?

Reply:
good idea!

Reply:
I removed this altogether instead of explaining it :)